starter/package.json

```diff
@@ -145,6 +145,7 @@
     "@types/core-js": "2.5.0",
     "@types/hammerjs": "2.0.35",
     "@types/node": "8.10.15",
+    "@types/webpack-env": "1.13.6",
```

Comment:
Could we move this dependency into Stark-Build since we have webpack dependencies into it ?

Reply:
Mmm in fact they are not related to Stark at all, I just replaced what was in custom-typings.d.ts on Showcase and Starter by this package since it is more up to date and complete. But as I said, they are not relevant for Stark, these are just a choice of the developer to include them.

Reply:
Ok, let's do that for now so